code/modules/mob/mob_defines.dm

```diff
@@ -54,6 +54,8 @@
 	var/eye_damage = 0//Living, potentially Carbon
 	var/lastpuke = 0
 
+	var/circulatory = 0 // Some simple mobs will receive damage from disabler bolts
```

Comment:
Make the adjustStaminaLoss() proc in simple_animal.dm apply damage(just copypaste from one of the others but replace the references to its damage type with stamina) and edit the  stamina damage coeff for these to 0.2